packages/@okta/vuepress-site/code/library-versions/index.md

```diff
@@ -63,10 +63,10 @@ This table lists the current (and historical) major versions of our open-source
 | [OIDC iOS][oidc-ios] (<i>formerly **iOS AppAuth SDK**</i>)      | 1.x<br>2.x<br>**3.x**                | Retired<br>Retired<br>**Stable**     |
 | [iOS OktaJWT][ios-oktajwt]                                      | 0.1.0<br>**1.x**                     | Retired<br>**Stable**                |
 | [iOS authentication SDK][authsdk-ios]                           | 0.1.0                                | Beta                                 |
-| [Java management SDK][sdk-java]                                 | 0.0.x<br>**1.x**                     | Retired<br>**Stable**                |
-| [Java authentication SDK][authsdk-java]                         | **1.x**                              | **Stable**                           |
+| [Java management SDK][sdk-java]                                 | 0.0.x<br>**3.x**                     | Retired<br>**Stable**                |
```

Comment:
Other libs (see OIDC-ios above) list all their versions, your edit skips to 3.x from 0.0.  Were the previous versions released?

Reply:
good point, updated.